src/main/cpp/Robot.cpp

```diff
@@ -21,7 +21,6 @@ void Robot::RobotInit() {
     intake.setGamepads(&gamepadOperator, &gamepadDriver);
 
     drivetrain.resetState();
-    drivetrain.setDriveMotorModeTo(rev::CANSparkMax::IdleMode::kCoast);
```

Comment:
This is still needed, but probably should be in teleop init

Reply:
I would like to argue that I don't think it is needed because the default is already coast